server/controllers/userController.js

```diff
@@ -50,3 +51,52 @@ exports.updateUserInfo = async function (req, res) {
     return res.status(400).json({ massage: err });
   }
 };
+
+// Update name, avaterUrl and timeZone
+exports.signUpUser = async function (req, res) {
+  const name = req.body.name;
+  const email = req.body.email;
+  const avatarUrl = req.body.avatarUrl;
+  let userId = "";
+
+  // Check user has account
+  const existUser = await Users.find({ email: email }, { email: 1 });
```

Comment:
It might be a dumb question on my side. But why are we checking for email:1 or does the line do something else.